Dockerfile.builder.tmpl

```diff
@@ -6,7 +6,7 @@ RUN apk add --no-cache \
 
 ENV XCADDY_VERSION v{{ .xcaddy_config.version }}
 # Configures xcaddy to build with this version of Caddy
-ENV CADDY_VERSION v{{ .config.caddy_version }}
+ARG CADDY_VERSION=v{{ .config.caddy_version }}
```

Comment:
If you want to do this, you should at least make this
```
ARG CADDY_VERSION=v{{ .config.caddy_version }}
ENV CADDY_VERSION=${CADDY_VERSION}
```
so that `xcaddy` will still see `CADDY_VERSION` as an env var.